service/version.go

```diff
@@ -0,0 +1,55 @@
+package service
+
+import (
+	"fmt"
+	"strings"
+
+	"golang.org/x/mod/semver"
+
+	"github.com/canonical/microcloud/microcloud/api/types"
+)
+
+const (
+	// lxdMinVersion is the minimum version of LXD that fully supports all MicroCloud features.
+	lxdMinVersion = "6.1"
```

Comment:
I think we still have to use `5.21` as this is the LTS.

Reply:
Oops, got too caught up in `.1` LTS syntax. Fixed now.